Web/starter_project_vanguard/components/layout/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import clsx from 'clsx'
+import Image from 'next/image'
+import Link from 'next/link'
+import { AiOutlineMenu } from 'react-icons/ai'
+import { navItems } from './data/nav-items'
+import { useState } from 'react'
+import logo from '../../public/img/a2sv-logo.png'
+
+
+const Navbar:React.FC = () => {
+  const [toggle, setToggle] = useState(false)
+  const [active, setActive] = useState("/")
```

Comment:
It is better if you can add the type.
`const [active, setActive] = useState<string>("/")` like this